components/ui-api-layer/internal/domain/servicecatalog/instance_resolver_test.go

```diff
@@ -431,10 +447,19 @@ func TestInstanceResolver_ServiceInstanceBindableField(t *testing.T) {
 
 		resolver := servicecatalog.NewInstanceResolver(instanceSvc, planGetter, classGetter)
 
-		parentObj := &gqlschema.ServiceInstance{
-			Name:             "test",
-			ServiceClassName: &className,
-			ServicePlanName:  &planName,
+		var parentObj *gqlschema.ServiceInstance
+		if it < 4 {
```

Comment:
It shouldn't be done like that, because if someone will change order or add some test data, then tests will fail and it will be confusing. That logic should be based on test data. You can add `className` and `planName` to test data and then such logic will be not needed.